conftest.py

```diff
@@ -196,6 +196,7 @@ def pytest_ignore_collect(collection_path, config):
             "tests/sentence_transformers",
             "tests/shap",
             "tests/sklearn",
+            "test/smolagents",
```

Comment:
```suggestion
            "tests/smolagents",
```

@y-okt This should fix the python/windows CI jobs

Reply:
Thank you!! Applied @B-Step62